Source/Fuse.Controls.Native/iOS/GraphicsView.uno

```diff
@@ -49,7 +50,10 @@ namespace Fuse.Controls.Native.iOS
 		[Foreign(Language.ObjC)]
 		static ObjC.Object CreateContainer()
 		@{
-			UIView* view = [[UIView alloc] init];
+			ContainerView* view = [[ContainerView alloc] init];
+			#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
```

Comment:
Since we already protect for iOS 13 inside the `viewDidLoad` method, protecting again here is redundant and may be confusing to future readers. We should probably drop the `#if...#endif` stuff here.

Reply:
Oh, I just thought of not calling a function to save CPU? 😄

Reply:
Done!

Reply:
Nice thought, but I doubt you're actually saving much CPU and this code is probably not very performance critical. Premature optimization is the root of all evil. ;)